examples/full-jaffle-shop-demo/dbt/models/metrics.yml

```diff
@@ -1,35 +0,0 @@
-version: 2
```

Comment:
could you rollback these  unrelated changes ? not sure if this is still needed somewhere...